packages/app/src/components/DBRowOverviewPanel.tsx

```diff
@@ -36,6 +36,17 @@ export function RowOverviewPanel({
   const resourceAttributes = firstRow?.__hdx_resource_attributes ?? EMPTY_OBJ;
   const eventAttributes = firstRow?.__hdx_event_attributes ?? EMPTY_OBJ;
 
+  const dataAttributes =
+    source.kind === 'log'
```

Comment:
hrm these attributes sholdn't exist for logs, at least not expected in the otel semantic conventions, is this necessary?

Reply:
You mean `LogAttributes`? I thought it was a part of the exporter schema?
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/clickhouseexporter/exporter_logs.go#L147

Reply:
I didn't look closely but things like `http.` attributes, but maybe it's called elsewhere and that's okay